src/client.test.ts

```diff
@@ -1,10 +1,26 @@
-import { queryApi } from './client';
-import browser from 'webextension-polyfill';
+import { queryApi } from "./client";
+import browser from "webextension-polyfill";
+import gql from "graphql-tag";
+import { TanagerMessageKey } from "./types";
 
-describe('queryApi', () => {
-  it('should send and message to the background script', async () => {
+describe("queryApi", () => {
```

Comment:
You're using double quotes?

Reply:
its auto-formatting to double quotes, I need to bring in the prettier config for single quotes formatting.